packages/cli/src/commands/account/new.ts

```diff
@@ -38,13 +39,23 @@ export default class NewAccount extends LocalCommand {
       description:
         "Language for the mnemonic words. **WARNING**, some hardware wallets don't support other languages",
     }),
+    mnemonic: flags.string({
+      description:
+        'Instead of generating a new mnemonic (seed phrases) the user can set the mnemonic to be used. It is required to set it as a string with all the words in order, separated by spaces (example: "word1 word2 word3 ... word24"). If the words are in other language than enaglish, the --language flag must be used. Should be a bip39 mnemonic',
+    }),
+    derivationPath: flags.string({
+      hidden: true,
+      description:
+        "Choose a different derivation Path (Celo's default is \"m/44'/52752'/0/0\"). This flags is hidden because is required only to help with specific problems (Example: A user that used in a transfer an Ethereum address and wants to recover its funds). NON technical users that don't understand the danger of changing it, SHOULDN'T use it",
```

Comment:
Typo? `m/44'/52752'/0'/0/`

Reply:
Given the idea is to make it simple for users to import from ethereum it might be nice to have flag like` —ethereumMnemonic` or special value for this flag like` —derivationPath=eth`

Reply:
Also Path->path

Reply:
I like --derivationPath=eth

Reply:
added